spec/core/ics-002-client-semantics/README.md

```diff
@@ -180,14 +205,48 @@ The `ConsensusState` MUST define a `getTimestamp()` method which returns the tim
 type getTimestamp = ConsensusState => uint64
 ```
 
-#### Header
```

Comment:
Removing `Header` should be made consistent throughout the entire standard (e.g., see the Update section).